components/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Image from "next/image";
+import logo from "../public/images/Logo.svg";
+import LoginBtn from "./utility-components/LoginBtn";
+function Navbar() {
+  return (
+    <div className=" fixed right-0 left-0 z-20 bg-[white] flex py-2 px-10 justify-between shadow-xl">
```

Comment:
![image](https://user-images.githubusercontent.com/45912180/200633835-1e15a060-a243-40bd-9c60-e1269e7532ca.png)
Navbar is covering the content on the page, please fix the position.